src/Agent.Listener/Program.cs

```diff
@@ -106,11 +109,27 @@ public async static Task<int> MainAsync(IHostContext context, string[] args)
                 var command = new CommandSettings(context, args, new SystemEnvironment());
                 trace.Info("Arguments parsed");
 
-                // Up front validation, warn for unrecognized commandline args.
-                var unknownCommandlines = command.Validate();
-                if (unknownCommandlines.Count > 0)
+                // Print any Parse Errros
+                if (command.ParseErrors != null && command.ParseErrors.Count() > 0)
```

Comment:
I have been told !(thing is null) is better than using !=

Reply:
I'm of the camp that `!=` is better unless you have a lot of existing code doing the latter. 

The history is that `is` was creating for testing object type and doing assignment slickly.

While yes you can use it for `null` it will do the same ` !=` check under the covers.

Reply:
I don't have a strong opinion. I think I picked that up from @vtbassmatt . Maybe Matt can tell us why?

Reply:
If you read the article I linked above, they are doing different things. And there is a case where != will fail the comparison, but only if you are overriding operators.

Reply:
Equality operator can be overridden, identity operator can't. In some other galaxy with much worse programmers, the equality check might not do what you think it does.

It's more of a habit thing that I force on myself, but in this instance doesn't really matter. I've also recently switched from `!(obj is null)` to `obj is object` which is easier to scan. (I'm pretty sure I picked this up from a StackOverflow topic by Jon Skeet, but I can't find it now.)

Reply:
this is a useful analysis: https://stackoverflow.com/a/45332827

Reply:
The examples here are no brainers to using `is`: 
https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-7#pattern-matching

Since this project already has signficantly more `!=` than `is` I'd prefer to stick with `!=` .  If someone ever did decide to overload operators we could have different behavior in different places.